Clears change note history together with deletion

elenatanasoiu opened the pull request:
Change note history does not get regenerated when you delete a change note and represent it downstream to content store.

This has to be cleared manually.

For: https://trello.com/c/sOPRBrrT/63-2-remove-change-note-and-redirect-html-attachment-url

thomasleese commented on db/migrate/20180216150252_remove_change_history_for_edition.rb:
```diff
@@ -0,0 +1,13 @@
+    edition.update!(details: edition_details)
+
+    if Rails.env.production?
+      Commands::V2::RepresentDownstream.new.call(edition.content_id)
```
You'll need to use `disable_ddl_transaction!` to use this.

thomasleese commented on db/migrate/20180216150252_remove_change_history_for_edition.rb:
```diff
@@ -0,0 +1,13 @@
+class RemoveChangeHistoryForEdition < ActiveRecord::Migration[5.1]
+  def change
+    edition = Edition.find(2975826)
```
Would be good to quit early if the `edition` doesn't exist.

elenatanasoiu commented:
Thanks @thomasleese
I've gone with your suggestion and removed change notes entirely. And added a check for edition. You caught me before I could add it. :)
